webclient/bower_components

```diff
@@ -0,0 +1 @@
+../bower_components/
```

Comment:
is this (and webclient/elements) a sym link? I personally would like to avoid using these. Especially since we can't run source anymore